api/v1beta2/foundationdbcluster_types.go

```diff
@@ -397,9 +397,12 @@ func (processGroupStatus *ProcessGroupStatus) AllAddressesExcluded(remainingMap
 
 	for _, address := range processGroupStatus.Addresses {
 		isRemaining, isPresent := remainingMap[address]
-		if !isPresent || isRemaining {
```

Comment:
Surely, it must be wrong to return that error if the address is present. Doesn't really matter for my code, though; I'd just need to reword the comment.